### ISSUE TITLE
When I delete model files (weight and json) why training accuracy change each time

### ISSUE THREAD

**Issue by ZakkiMasood** [EnJiang/SHU-CAPTCHA-solver-based-on-CNN#1]

**Comment by EnJiang** [comment]
Sorry for the late reply.
Not sure what you mean... if you what ran is train.py, the information printed in the console is the current accuracy of the model, this python file will not load model files at all, it starts a whole new model, so you can see the model improves epoch by epoch. See train.py and you can see after given epochs, the code stops and store the model files.

**Comment by ZakkiMasood** [comment]
Thanks for your positive response, yes you are right.

One thing more I stuck with the ACCURACY GRAPH, how I will make after
training the network to make accuracy graph from model files. I tried at my
end below is code, Is it fine?

Sorry for disturbing you again because I am new in it, Please...

from keras.optimizers import SGD
import numpy as np
import matplotlib.pyplot as plt
from keras.models import model_from_json

X_train = np.load("X_tain.npy")
Y_train = np.load("Y_train.npy")


# load json and create model
json_file = open('shu_captcha_CNN_structure.json', 'r')
loaded_model_json = json_file.read()
json_file.close()
loaded_model = model_from_json(loaded_model_json)

# load weights into new model
loaded_model.load_weights("shu_captcha_CNN_weights.h5")
print("Loaded model from disk")

sgd = SGD(lr=1e-5, decay=0, momentum=0.9, nesterov=True)
loaded_model.compile(loss='categorical_crossentropy', optimizer=sgd,
metrics=['accuracy'])
history = loaded_model.fit(X_train,  Y_train, validation_split=0.1,
epochs=500, batch_size=32, verbose=1)







*plt.plot(history.history['acc']) plt.plot(history.history['val_acc'])
plt.title('model
accuracy') plt.ylabel('accuracy') plt.xlabel('epoch') plt.legend(['train',
'test'], loc='upper left') plt.show() *







*plt.plot(history.history['loss'])
plt.plot(history.history['val_loss']) plt.title('model
loss') plt.ylabel('loss') plt.xlabel('epoch') plt.legend(['train', 'test'],
loc='upper left') plt.show()*

**Comment by EnJiang** [comment]
You used "history = loaded_model.fit(...)" to get the current training status, I'm not sure if this works. Maybe you should check keras api for details.
https://keras.io/
